server/src/test/java/io/spine/examples/shareaware/server/given/GivenMoney.java

```diff
@@ -19,4 +19,22 @@ public static Money zero() {
                 .setNanos(0)
                 .vBuild();
     }
+
+    public static Money generatedWith(long units, Currency currency) {
```

Comment:
Let's rename these "generated" methods to speak about money. Later on, these methods may be imported statically.